tests/storage/hotplug.go

```diff
@@ -122,6 +123,7 @@ var _ = SIGDescribe("Hotplug", func() {
 					},
 				},
 				Serial: volumeName,
+				Cache:  v1.DriverCache(cache),
```

Comment:
Do you want to add it here if `cache == ""`?

Reply:
> Do you want to add it here if `cache == ""`?

I think `cache == ""` is illegal, an error message should be returned.